packages/server/src/seeder/utils.ts

```diff
@@ -313,4 +333,33 @@ export async function addLikesToArticles(_min: number, _max: number): Promise<vo
     // 5. first half of articles will be like by a user
     // 6. remaining half of articles will be dislike by a user
     // Perform the above task in parallel ( Promise.all() ) for each and every user
+
+    const prisma = PrismaClientSingleton.prisma;
+
+    const [articles, users] = await Promise.all([await prisma.article.findMany(), await prisma.user.findMany()])
+
+    // calculates the number for 60% of all articles
+    const percentage = Math.ceil(articles.length * 0.6);    
+
+    // Shuffles and then populates up to the percentage number of articles
+    const selectedArticles = articles.sort(() => 0.5 - Math.random()).slice(0, percentage);
```

Comment:
Try to come up with meaningful variable names like randomSelectedArticles or something. It is okay for now.